src/routes/device.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import DeviceService from '../services/DeviceService';
+
+const router: express.Router = express.Router();
+
+router.post('/', (req, res) => {
+  DeviceService.addDevice(req.body.mail, req.body.token);
```

Comment:
Catch block is missing, which causes errors to be elevated into express.js. Respond with an HTTP 400 if something goes wrong.